docs/reference/configuration.md

```diff
@@ -100,6 +100,10 @@ This reference uses `.` to denote the nesting of values.
 * `auth.oidc.authorize_endpoint_query_parameters` `(map[string]string : )` - Add these parameters when calling the provider's `/authorize` endpoint
 * `auth.oidc.validate_id_token_claims` `(map[string]string : )` - When a user tries to access lakeFS, validate that the ID token contains these claims with the corresponding values.
 * `auth.oidc.additional_scope_claims` `(string[]: [])` - Add these additional scopes/claims to the OIDC user token. `openid` and `profile` are provided already. 
+* `auth.ui_config.RBAC` `(string: "simplified")` - "simplified" or
```

Comment:
I wonder if this can be associated with the external auth configuration instead of maintaining a couple of config params

Reply:
It can, but I would rather not.  The upcoming RBAC removal also makes this distinction.  It can make sense to provide an external auth server and still not provide RBAC in the GUI.  For instance when the external auth server might be shared between multiple lakeFS installations.